pom.xml

```diff
@@ -1,383 +1,376 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.1.4.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
 
-	<groupId>org.rulez.demokracia</groupId>
-	<artifactId>PDEngine</artifactId>
-	<version>0.0.1-feature_java10_and_formatting-afterDocker.24ef1d3</version>
-	<packaging>war</packaging>
-
-	<name>PDEngine</name>
-	<url>http://maven.apache.org</url>
-
-	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.source>11</maven.compiler.source>
-		<maven.compiler.target>11</maven.compiler.target>
-		<sonar.github.repository>edemo/PDEngine</sonar.github.repository>
-		<xmldoclet-version>1.0.6-SNAPSHOT</xmldoclet-version>
-	</properties>
+  <groupId>org.rulez.demokracia</groupId>
+  <artifactId>PDEngine</artifactId>
```

Comment:
It seems that the org.rulez.demokracia.persistency java package is a leftover. I guess it should be deleted.
Commented here in need of a place to comment it to.